browser/ui/webui/brave_federated/federated_internals.mojom

```diff
@@ -5,12 +5,13 @@
 
 module federated_internals.mojom;
 
-struct AdStoreLog {
-    int32 log_id;
-    double log_time;
-    string log_locale;
-    int32 log_number_of_tabs;
-    bool log_label;
+struct Covariate {
```

Comment:
I think federated internals should import `brave_federated.mojom`

Reply:
👌 some of this stuff is still provisional. Had some weird linking problem when trying to import brave_federated.mojom so I circumvented it by replicating it

Reply:
👌 some of this stuff is still provisional. Had some weird linking problem when trying to import brave_federated.mojom so I circumvented it by replicating it